### ISSUE TITLE
avgpool stride confusion in ResNet-50

### ISSUE THREAD

**Issue by ozancaglayan** [pytorch/vision#304]
Hello,

The default value `None` of `stride` for avgpool methods imposes a stride which is equal to `kernel_size` leading to textual representations like below:

```
(avgpool): AvgPool2d (size=7, stride=7, padding=0, ceil_mode=False, count_include_pad=True)
```

Although a stride of 7 or 1 in this case does not change anything in the output (since the input tensor's spatial resolution is 7x7), I had confusion when comparing PyTorch architecture to [caffe proto file](https://github.com/KaimingHe/deep-residual-networks/blob/master/prototxt/ResNet-50-deploy.prototxt#L2299) where the stride is explicitly given as 1. So I think it would be better to explicitly pass `stride=1` to ResNet's last `avgpool` to make things clear. What do you think?

**Comment by alykhantejani** [comment]
Hi @ozancaglayan,

While either stride `7` or `1` in this case, after discussing with @fmassa he also prefers if we use a stride of `1` to be consistent with other frameworks when representing/printing these models. PR #306 should fix this once merged.

**Comment by alykhantejani** [comment]
Fixed in #306
